geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreParams.scala

```diff
@@ -23,114 +23,135 @@ object HBaseDataStoreParams extends GeoMesaDataStoreParams with SecurityParams {
       "Catalog table name",
       optional = false,
       deprecatedKeys = Seq("bigtable.table.name"),
-      supportsNiFiExpressions = true)
+      supportsNiFiExpressions = true
+    )
 
   val ConnectionParam =
     new GeoMesaParam[Connection](
       "hbase.connection",
       "Connection",
-      deprecatedKeys = Seq("connection"))
+      deprecatedKeys = Seq("connection")
+    )
 
   val ZookeeperParam =
     new GeoMesaParam[String](
       "hbase.zookeepers",
       "List of HBase Zookeeper ensemble servers, comma-separated. " +
           "Prefer including a valid 'hbase-site.xml' on the classpath over setting this parameter",
-      supportsNiFiExpressions = true)
+      supportsNiFiExpressions = true
+    )
 
   val CoprocessorUrlParam =
     new GeoMesaParam[Path](
       "hbase.coprocessor.url",
       "URL pointing to the GeoMesa coprocessor JAR",
       deprecatedKeys = Seq("coprocessor.url"),
-      supportsNiFiExpressions = true)
+      supportsNiFiExpressions = true,
+      readWrite = ReadWriteFlag.WriteOnly
```

Comment:
This one takes a second to realize why it should be a 'write' param.   Hmm... configuring HBase is tough.:(